### PR TITLE
Allow downcase of column names and all Req options

### PR DESCRIPTION
**Pull request by forest**

### CODE REVIEW

File: lib/avalanche.ex [hunk 1]
```diff
@@ -227,10 +228,26 @@ defmodule Avalanche do
     end
   end
 
+  @doc """
+  List of available Req request options.
+
+  See `Req.request/1` for more information.
+  """
+  @spec available_req_options :: list(atom())
+  def available_req_options do
```
Comment (pdgonzalez872) [inline lib/avalanche.ex:237]: does `Req` expose these? Seems like this could potentially be hard to maintain?

Comment (pdgonzalez872) [reply lib/avalanche.ex:237]: just a nit

**Review by pdgonzalez872 — APPROVED** [review]

File: lib/avalanche.ex (hunk 1 above)
Comment (forest) [reply lib/avalanche.ex:237]: I don't think we can get them dynamically from `Req`. Yes, kinda a pain to maintain.
https://hexdocs.pm/req/Req.html#request/1-options

The list is copied from here: https://github.com/wojtekmach/req/blob/main/lib/req.ex#L93

**Pull request merged**